dist/obs-bundled-gems.spec

```diff
@@ -42,6 +42,7 @@ BuildRequires:  make
 BuildRequires:  mysql-devel
 BuildRequires:  nodejs
 BuildRequires:  python-devel
+BuildRequires:  ImageMagick-devel
```

Comment:
I'm not so sure we want to introduce this dependency. ImageMagick has not the greatest [track record regarding security](https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=imagemagick). Especially for releases.

Reply:
https://caniuse.com/svg looks pretty good right? Do we need png?

Reply:
Sure, that still leaves calculating width of text for rendering svg though

Reply:
> calculating width of text for rendering svg though

Why would we need rmagick for that?

Reply:
Calculating width of text otherwise is pretty complex, because rendering text in the first place is pretty complex. We could use per-character font widths based on ttf, but then this means compound glyphs aren't measured properly (technically in case of latin alphabet that works out, since we will in most cases have more space than needed than less space)